lib/views/chatroom/chatroom_components/reaction_bar.dart

```diff
@@ -0,0 +1,98 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_emoji/flutter_emoji.dart';
+import 'package:group_chat_example/utils/ui_utils.dart';
+
+import '../enums/reaction_enum.dart';
+import 'reaction_button.dart';
+
+class ReactionBar extends StatelessWidget {
+  final List reactions;
+  final Function() refresh;
+  final EmojiParser emojiParser = EmojiParser();
```

Comment:
We should never instantiate an object directly in the class, if we do so, we will not be able to test this class. Instead, pass the EmojiParser as an argument.

Reply:
Done